### PR TITLE
docs: Update installation instructions for the monorepo changes

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé

Updating the installation instructions in the repo's main README to reflect the changes for the mono repo setup.

Translations provided by @EliseKa (big thank you!)

To preview, you can visit https://github.com/cds-snc/gcds-components/tree/docs/monorepo-instructions and scroll down to see how it looks like.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,11 +78,15 @@ Toute la documentation sur les composants de Système de design GC est accessibl
 ## Installation locale
 
 - Copiez le référentiel `git clone https://github.com/cds-snc/gcds-components`.
-- Allez dans le répertoire `/packages/web`.
 - Exécutez ensuite `npm install` pour installer toutes les dépendances Node.js.
-- Finalement, exécutez `npm run build` pour compiler les composants Web.
+- Finalement, exécutez `npm run build` pour compiler les composants Web. [FR]
+- Finalement, exécutez `npm run build` pour compiler les trois paquets (composants web, react et angular). [FR]
```
Comment (EliseKa) [inline README.md:82]: Finalement, exécutez `npm run build` pour compiler les trois paquets (composants Web, React et Angular).

(it was alright, I only added capital letters)

**Review by EliseKa — COMMENTED** [review]
Providing the translations for the README copy changes

File: README.md (hunk 1 above)
Comment (daine) [reply README.md:82]: Thank you, appreciate it!

Comment (daine) [reply README.md]: Line 82 (Exécutez npm run build pour compiler les composants Web) will be removed and replaced with new text including three packages.

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Review by melaniebmn — APPROVED** [review]
LGTM

**Pull request merged**